IssueTriage.md

```diff
@@ -0,0 +1,103 @@
+# Triage Guidance for WebNN Repo
+
+TODO: Consider removing these, or fit these into the documentation below.
+
+- **enhancement** - unclear if this is about the spec text, the proposed API, etc. Prefer **feature request** etc.
+- **help wanted** - unclear what help is needed; use **needs PR** or **question** instead?
+
+
+## Label Usage
+
+Labels are used for:
+
+- Understanding the status of a specific issue and next steps to resolve it.
+- Understanding the scope of work remaining on broad efforts (e.g. aligning with best practices, fixing normative issues, etc)
+- Identifying areas to contribute.
+
+The working group chairs and spec editors should regularly review bugs and ensure that labels are accurate, and ensure that issues are getting appropriate attention; for example, scheduling discussion of new feature requests, discussion to resolve outstanding questions, and drawing attention to issues that are ready for a contributor to author a PR.
+
+
+## Types
+
+Every issue should have one of these issue types, and only rarely more than one.
+
+- **bug** - a flaw in the specification that will require a normative fix; for example, an algorithm computes an output incorrectly
+- **conventions** - where the spec does not conform to specification best practices from Web IDL, Bikeshed, Infra, etc.
+- **use case** - a new use case for the API that should be documented or considered; may spawn other issues
+- **process** - a meta issue about how the specification is evolved; for example, a discussion of issue labels
+- **testing** - discussion of test coverage
+- **feature request** - _PROPOSED_ - suggestion for an addition to the proposed API
```

Comment:
Added https://github.com/webmachinelearning/webnn/labels/feature%20request to test drive this.

Reply:
I did a pass and labeled issues that looked like requests for pure functionality additions. How does it look to others?

Reply:
> I'll defer to others who know the space better than I do re: "opset" vs. "op set" vs. "operator set" etc.

I've always just said `opset` as a shorthand for `operator set`. 🤷‍♂️

Reply:
`opset` for label sgtm.

Reply:
Updated proposal w/ `opset`

Reply:
Renamed to https://github.com/webmachinelearning/webnn/labels/opset